packages/client/tests/functional/invalid-env-value/tests.ts

```diff
@@ -0,0 +1,58 @@
+import stripAnsi from 'strip-ansi'
```

Comment:
This is actually a rename of the test below with some added test cases for each type of engine/runtime.